tests/devapps/XForms/XForms/TestPage.xaml.cs

```diff
@@ -311,7 +311,13 @@ private void CreateExceptionMessage(Exception exception)
                 acquireResponseLabel.Text = "Exception - " + exception.Message;
             }
 
-            Console.WriteLine(exception);
+            Console.WriteLine(string.Format(CultureInfo.InvariantCulture, "Exception -\nMessage: {0}\nStack Trace: {1}",
```

Comment:
If you just replace the entire block with 

```csharp
Console.WriteLine(exception);
```

it will be enough. The exception.ToString() method takes care of outputting the message, the stack trace and the inner exception details